data/parser/parser_suite_test.go

```diff
@@ -5,9 +5,136 @@ import (
 	. "github.com/onsi/gomega"
 
 	"testing"
+
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/log"
+	"github.com/tidepool-org/platform/service"
 )
 
 func TestSuite(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "data/parser")
 }
+
+type AppendErrorInput struct {
```

Comment:
ahh OK had never thought of adding all this in the `suite` but makes the tests much cleaner and focused